lib/Interpreter/CppInterOp.cpp

```diff
@@ -3161,6 +3161,69 @@ namespace Cpp {
     return PI->getNameAsString();
   }
 
+  TCppFunction_t GetBinaryOperator(TCppScope_t scope, const std::string& op,
+                                   const std::string& lc,
+                                   const std::string& rc) {
+    Decl* D = static_cast<Decl*>(scope);
+    if (!D)
+      return nullptr;
+
+    DeclContext* DC = D->getDeclContext();
+    if (!DC)
+      return nullptr;
+
+    Scope* S = getSema().getScopeForContext(DC);
+    if (!S)
+      return nullptr;
+
+    clang::UnresolvedSet<8> lookup;
+
+    if (op == "+")
+      getSema().LookupOverloadedOperatorName(clang::OO_Plus, S, lookup);
```

Comment:
Looks like `Sema::LookupBinOp` is a higher-level interface. We should probably use it instead...